src/common/SPELLS/rogue.js

```diff
@@ -476,8 +476,22 @@ export default {
     name: 'Blade Flurry',
     icon: 'ability_warrior_punishingblow',
   },
-
+  BLADE_RUSH_DAMAGE:{
```

Comment:
I noticed that BladeRush Energy spell is missing and causing an error in the energy tab. Cam you also add that please.